### ISSUE TITLE
Parameters on DeskewGui v0.90

### ISSUE THREAD

**Issue by vivadavid** [galfar/deskew#3]
Hello,

After doing some minor testing with the default and the lanczos filters, I agree that the quality under lanczos is noticeably better, and in my old computer, it doesn't take much longer to process (I would say that a couple of seconds per page).

In conclusion, I'd like to use lanczos from now on, but DeskewGui doesn't allow me, to the best of my knowledge, to include parameters when I call deskew.exe.

Is there way I can tell the programme that I want to use lanczos?

Thanks!

**Comment by galfar** [comment]
Hi, I've just added selection of resampling filter to Deskew GUI (Adv. Options) and upped the version to 0.92. Not sure when I get to do new binary release though.

**Comment by vivadavid** [comment]
Thank you, @galfar !

That's excellent news! I'll be happy to try the new Deskew GUI when it's ready.

**Comment by vivadavid** [comment]
Hello, @galfar,

I don't want to bother you, but I'd like to know if you are planning to release the new binary soon.

Meanwhile, I could use the command line, but I couldn't find how to batch process. So far, I've created a link to the file "deskew.exe" and, on this link, I've added the following parameters:

deskew -o out.png -b FFFFFF -q lanczos image.png

If I drag a file (from any folder) into the link (and the file doesn't need to be called "image.png"), an aligned image is created, but if I drag several files, only one of them is processed. I'd appreciate some help.

Thanks!
